app/graphql/resolvers/generic_item_search.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "search_object/plugin/graphql"
+require "graphql/query_resolver"
+
+class Resolvers::GenericItemSearch
+  include SearchObject.module(:graphql)
+
+  scope { GenericItem.all }
+
+  type types[Types::QueryTypes::GenericItemType]
+
+  class GenericItemOrder < ::Types::BaseEnum
+    value "createdAt_ASC"
+    value "createdAt_DESC"
+    value "updatedAt_ASC"
+    value "updatedAt_DESC"
+    value "id_ASC"
+    value "id_DESC"
+  end
+
+  option :limit, type: types.Int, with: :apply_limit
+  option :skip, type: types.Int, with: :apply_skip
+  option :ids, type: types[types.ID], with: :apply_ids
+  option :order, type: GenericItemOrder, default: "createdAt_DESC"
```

Comment:
```
option :dataProvider, type: types.String, with: :apply_data_provider
option :dataProviderId, type: types.ID, with: :apply_data_provider_id
option :categoryId, type: types.ID, with: :apply_category_id
option :contentType, type: types.String, with: :apply_content_type
option :externalId, type: types.ID, with: :apply_external_id
```

siehe #191

Reply:
erweitert in https://github.com/ikuseiGmbH/smart-village-app-mainserver/pull/190/commits/bc7856e67c753400aa9db567d26eb2d71e87eab3